From 323406346e7da45e90d997a2b94ac1b08d4d1ab7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 6 Sep 2026 03:05:47 +0800
Subject: [PATCH] AOSCOS: fix(color-schemes/Breeze): use bright/bold white for
 ForegroundIntense

Who came up with that light-blue-is-intensive white design anyway...

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 data/color-schemes/Breeze.colorscheme | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/color-schemes/Breeze.colorscheme b/data/color-schemes/Breeze.colorscheme
index fac9eff61..b685ed87f 100644
--- a/data/color-schemes/Breeze.colorscheme
+++ b/data/color-schemes/Breeze.colorscheme
@@ -86,7 +86,7 @@ Color=252,252,252
 Color=239,240,241
 
 [ForegroundIntense]
-Color=61,174,233
+Color=255,255,255
 
 [General]
 Description=Breeze
-- 
2.55.0

